www/download.php

```diff
@@ -241,7 +241,7 @@ function downloadSingleFile($transfer, $recipient, $file_id, $recently_downloade
         if (!$chunk_size)
             $chunk_size = 1024 * 1024;
 
-        if($file->encrypted_size){
+        if($file->transfer->options->encryption){
```

Comment:
`$file->encrypted_size` is set even if `$file->transfer->options->encryption`. Maybe it would be a good idea to _not_ calculate `$file->encrypted_size` if no encryption is used. This would need to be changed in Transfer.class.php.